CodeStarterCamp_Week2/main.swift

```diff
@@ -8,5 +8,29 @@
 
 import Foundation
 
-print("Hello, World!")
+//Step 2
+func makeLottoNumbers() -> [Int] {
+    var lottoNumbers: Set<Int> = []
+    while lottoNumbers.count < 6,
+          let randomInt: Int = (1...45).randomElement() {
+        lottoNumbers.insert(randomInt)
+    }
+    return Array(lottoNumbers)
+}
```

Comment:
```suggestion
func makeLottoNumbers() -> [Int] {
    var lottoNumbers: Set<Int> = []
    
    while lottoNumbers.count < 6,
          let randomInt: Int = (1...45).randomElement() {
        lottoNumbers.insert(randomInt)
    }
    
    return Array(lottoNumbers)
}
```

- 가독성이 좋도록 개행이 있으면 좋을 거 같아요😊
- `Set`로 변수를 설정하시고, `Array`로 반환하는 데 이유가 있으실까요?

Reply:
다른 이유는 없고 Step2 Todo에 '내가 찍은 번호 배열을 myLottoNumbers 변수에 할당합니다.' 라는 문구를 보고 Array로 반환했습니다!

Reply:
그리고 가독성 있게 띄었습니다!